.github/workflows/ci.yml

```diff
@@ -26,3 +27,85 @@ jobs:
     - run: make lint
     - run: make test
 
+  docker:
+    needs: [test]
+    # if: startsWith(github.ref, 'refs/heads/bump-version-') || startsWith(github.ref, 'refs/tags/v')
```

Comment:
Maybe the execution condition of each job/step will be clearer if `if` is placed right under the name.